secretstores/huaweicloud/csms/csms.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package csms
+
+import (
+	"github.com/huaweicloud/huaweicloud-sdk-go-v3/core/auth/basic"
+	csms "github.com/huaweicloud/huaweicloud-sdk-go-v3/services/csms/v1"
+	"github.com/huaweicloud/huaweicloud-sdk-go-v3/services/csms/v1/model"
+	csmsRegion "github.com/huaweicloud/huaweicloud-sdk-go-v3/services/csms/v1/region"
+
+	"github.com/dapr/components-contrib/secretstores"
+	"github.com/dapr/kit/logger"
+)
+
+const (
+	region          string = "region"
+	accessKey       string = "accessKey"
+	secretAccessKey string = "secretAccessKey"
+	pageLimit       string = "100"
+	latestVersion   string = "latest"
+	versionId       string = "version_id"
```

Comment:
Same comment as on your original PR

Are you sure you want to name the metadata field like this?

We do not use `_` in metadata property names generally.

I recommend you make this `versionId`

Reply:
Never mind -- I see this is not a component metadata but a request metadata field. That's fine then!